Updated: what is vanus (#248)
linkall-labs/docs: content/introduction/what-is-vanus.md

title: What is Vanus
id: what-is-vanus
---

> **Vanus is an open-source serverless event streaming platform with built-in event processing capabilities.**  It connects SaaS, cloud services, and databases to help users build next-generation event-driven applications. Vanus separates storage and computing resources and offers modern development features such as CloudEvents native, FaaS Integration, built-in Connectors, data filtering, and transformation, etc. 

Reviewer: ```suggestion
**Vanus is an open-source serverless event streaming platform with built-in event processing capabilities.**  It connects SaaS, cloud services, and databases to help users build next-generation event-driven applications. Vanus separates storage and computing resources and offers modern development features such as CloudEvents Specification, FaaS Integration, built-in Connectors, data filtering, and transformation, etc. 
```

> **Vanus is an open-source serverless event streaming platform with built-in event processing capabilities.**  It connects SaaS, cloud services, and databases to help users build next-generation event-driven applications. Vanus separates storage and computing resources and offers modern development features such as CloudEvents Specification, FaaS Integration, built-in Connectors, data filtering, and transformation, etc. 

Reviewer: ```suggestion
**Vanus is an open-source serverless event streaming platform with built-in event processing capabilities.**  It connects SaaS, cloud services, and databases to help users build next-generation event-driven applications. Vanus separates storage and computing resources and offers modern development features such as CloudEvents Specification, FaaS Integration, built-in Connectors, data filtering, and transformation.

#### Build the event-driven application
* Send SaaS-generated events to the data lake for analysis.
* Deliver cloud services events to cloud functions for processing.
* Real-time transmission of events between SaaS.
* Synchronize data between databases in real-time.

#### Out-of-the-box event computing capabilities
* Provides 100+ built-in functions to help developers process events in real-time.
* Provides general and flexible filtering rules, and developers can easily filter events.
* Supports event processing through cloud functions such as aws lambda.

> #### Serverless, super easy to use

Reviewer: ```suggestion
#### Serverless, a simple and effortless process.
```

* Automatically scale up or down clusters based on event traffic, reducing costs by up to 90%.
* Seamlessly integrate mainstream cloud functions and open-source FaaS platforms.

> * One-click deployment, the installation is completed within 1 minute, 0 operation

Reviewer: ```suggestion
* One-click deployment, the installation is done in seconds with 0 operations needed.
```

> Vanus is fully an Open Source project, which is publicly available under the Apache 2.0 License. You can find Vanus on [GitHub](https://github.com/linkall-labs/vanus).

Reviewer: ```suggestion
Vanus is an Open-Source project publicly available under the Apache 2.0 License. You can find Vanus on [GitHub](https://github.com/linkall-labs/vanus).
```

> Vanus is an Open-Source project publicly available under the Apache 2.0 License. You can find Vanus on [GitHub](https://github.com/linkall-labs/vanus).

Reviewer: ```suggestion
Vanus is an open-source project publicly available under the Apache 2.0 License. You can find Vanus on [GitHub](https://github.com/linkall-labs/vanus).
```
